### DISCUSSION TITLE
I do not want a social media-esque feed

### DISCUSSION THREAD

**Discussion by Dekkonot** [community/community#55081] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I am asking you politely to not push a feed onto me that includes things like "X followed Y" because I genuinely could not care less about that information, especially in cases where I am not following either person.

I do not want to see "trending repositories". I do not want to know when people fork repositories or anything of that sort. I care about repositories I regularly interact with and topics I'm involved in. Nothing else.

**Comment by cornfeedhobo** [comment]
This was the first title that matches my reaction to the new feed. I _really_ dislike it. I don't care that GPT is the new hotness - I'm 1000% not interested in it, yet my entire feed is filled with crap related to this pos fad.

Microsoft, I like a lot of the changes you are making, but this is a terrible idea. Reverse course!

**Comment by deadjakk** [comment]
yeah I don't like this at all, I much prefer the older, simpler feed.
